x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -155,6 +163,48 @@ export const TransformManagement: FC = () => {
             )}
             {typeof errorMessage === 'undefined' && (
               <AlertRulesManageContext.Provider value={getAlertRuleManageContext()}>
+                {transformIdsWithoutConfig ? (
+                  <>
+                    {deleteAction.isModalVisible && (
+                      <DeleteActionModal {...deleteAction} hasNoConfig={true} />
+                    )}
+
+                    <EuiCallOut color="warning">
+                      <p>
+                        <FormattedMessage
+                          id="xpack.transform.danglingTasksError"
+                          defaultMessage="Found {count} {count, plural, one {transform} other {transforms}} [{transformIds}] with no corresponding  {count, plural, one {configuration} other {configurations}}."
+                          values={{
+                            count: transformIdsWithoutConfig.length,
+                            transformIds: transformIdsWithoutConfig.join(', '),
+                          }}
+                        />
+                      </p>
+                      <EuiButton
+                        color="warning"
+                        size="s"
+                        onClick={() =>
+                          deleteAction.openModal(
+                            transformIdsWithoutConfig.map((id) => ({
+                              id,
+                              config: undefined,
+                              stats: undefined,
+                            }))
+                          )
+                        }
+                      >
+                        <FormattedMessage
+                          id="xpack.transform.forceDeleteTransformMessage"
+                          defaultMessage="Force delete {count} {count, plural, one {transform} other {transforms}}"
```

Comment:
Do users really need to care that we're "force-deleting"? I couldn't find a lot of other hits for messages that add that clause, so perhaps this is sufficient:
```suggestion
                          defaultMessage="Delete {count} {count, plural, one {transform} other {transforms}}"
```

Reply:
Agree that we should drop the term `Force` here.

Reply:
Removed here https://github.com/elastic/kibana/pull/123407/commits/3c0a8c0048f129cb3e1cbdba81fc1b54616e6685